kernel_config

```diff
@@ -1293,10 +1293,17 @@ CONFIG_BAYCOM_SER_FDX=m
 CONFIG_BAYCOM_SER_HDX=m
 CONFIG_BAYCOM_PAR=m
 CONFIG_YAM=m
-# CONFIG_CAN is not set
 CONFIG_IRDA=m
 
 #
+# CAN drivers
+#
+CONFIG_CAN=y
+CONFIG_CAN_RAW=y
+CONFIG_CAN_BCM=y
+CONFIG_CAN_VCAN=y
```

Comment:
I'm OK with the general idea, but I think we should do `=m` instead of `=y`, similar to Debian: https://sources.debian.net/src/linux/stretch/debian/config/config/#L6388-L6394

```
##
## file: net/can/Kconfig
##
CONFIG_CAN=m
CONFIG_CAN_RAW=m
CONFIG_CAN_BCM=m
CONFIG_CAN_GW=m
```

Can you think of a reason why they don't set `CONFIG_CAN_VCAN`, and instead set `CONFIG_CAN_GW`?  Any reason we shouldn't simply match what they're doing?  (Debian's certainly got more manpower behind their kernel maintenance than we do in b2d)